### PR TITLE
Marcgabe15/health route

### PR DESCRIPTION
**Pull request by marcgabe15**
# Description
This PR adds in health route and test cases for it

# Related Issue
N/A will make issue

# Motivation and Context
we need a general workflow for routes and controllers

# How Has This Been Tested?
controller route in __tests__ and has 100% coverage

# Screenshots (if appropriate):

### CODE REVIEW

File: __tests__/health.controller.test.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+describe('GET /v1/health', () => {
+  beforeAll((done: jest.DoneCallback) => {
+    done();
+  });
+
+  afterAll((done: jest.DoneCallback) => {
+    done();
+  });
+  it('health api request', async () => {
```
Comment (HamishPierpont) [inline __tests__/health.controller.test.ts:12]: Small nitpick, but I think the first parameter to `it(` should be a verb. So this should be `it('gets an OK response from the health api', ...` or something similar.

**Review by HamishPierpont — CHANGES_REQUESTED** [review]
LGTM, except for one small thing with the health controller test.

**Review by dhruv2000 — APPROVED** [review]
LGTM

**Review by HamishPierpont — APPROVED** [review]
LGTM

**Review by HamishPierpont — APPROVED** [review]
LGTM

**Review by dhruv2000 — APPROVED** [review]
LGTM

**Pull request merged**